File: Praktikum2/Java/Hardware.java
class Hardware extends Product {
    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getBrand() {
        return this.brand;
    }

    public void setModel(String model) {
        this.model = model;
    }

    public String getModel() {
        return this.model;
    }

    public void printHardware() {
        System.out.println("Brand : " + this.brand);
        System.out.println("Model : " + this.model);
    }
}

File: Praktikum2/C++/Hardware.cpp
#include <bits/stdc++.h>
#include "Product.cpp"
using namespace std;

class Hardware : public Product
{
private:
    string brand;
    string model;

public:
    Hardware() {}

    void setBrand(string brand)
    {
        this->brand = brand;
    }

    string getBrand()
    {
        return this->brand;
    }

    void setModel(string model)
    {
        this->model = model;
    }

    string getModel()
    {
        return this->model;
    }

    void printHardware()
    {
        cout << "Brand : " << getBrand() << endl;
        cout << "Model : " << getModel() << endl;
    }

    ~Hardware() {}
};